File: Python/1040.py
x = input().split()
n1,n2,n3,n4 = x
media = (float(n1) * 2 + float(n2) * 3 + float(n3) * 4 + float(n4) * 1) / 10
print("Media: %0.1f"%media)
if media >= 7:
    print('Aluno aprovado.')
elif media < 5:
    print('Aluno reprovado.')
if 5.0 <=  media <= 6.9:
    print('Aluno em exame.')
    nExame = float(input())
    print("Nota do exame: %0.1f"% nExame)
    nFinal = (nExame + media) / 2
    if nFinal >= 5:
        print('Aluno aprovado.')
        print("Media final: %0.1f" % nFinal)
    else:
        print('Aluno reprovado.')
        print("Media final: %0.1f" % nFinal)

File: JavaScript/1040.js
let input = require('fs').readFileSync('./dev/stdin/file.txt', 'utf8')
let = lines = input.split('\n')
const pegarValores = (line) => line.split(" ").map(a => +(a))

const [notaA, notaB, notaC, notaD] = pegarValores(lines.shift())
media = ((notaA * 2) + (notaB * 3) + (notaC * 4) + (notaD * 1)) / 10
console.log(`Media: ${media.toFixed(1)}`)
if (media >= 7) {
    console.log('Aluno aprovado.')
} else if (media < 5) {
    console.log('Aluno reprovado.')
} else {
    let notaExame = +lines.shift()
    console.log('Aluno em exame.')
    console.log(`Nota do exame: ${notaExame.toFixed(1)}`)
    let mediaExame = (notaExame + media) / 2
    if (mediaExame >= 5) {
        console.log('Aluno aprovado.')
    } else {
        console.log('Aluno reprovado.')
    }
    console.log(`Media final: ${mediaExame.toFixed(1)}`)
}
